pages/documentation/simulation/shortcircuitanalysis/index.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: default
+---
+
+# Short circuit API
+
+* TOC
+{:toc}
+
+## Introduction
+When a short circuit occurs in a network, the currents on equipment can be so high that they exceed their rated values.
+Simulating faults on the network is important to verify that the short circuits are well detected and do not damage the equipments.
+
+The short circuit API allows to compute the currents and voltages on a network after a fault. 
+For the moment, no simulator is available to compute the analysis, but it is possible to plug one to this API.
+
+
+## Parameters
+
+### Available parameters
+The parameters to be used for the short circuit calculation should be defined in the config.yml file. For example, here are some valid short circuit parameters:
+
+```yaml
+short-circuit-parameters:
+  with-voltage-map: false
+  with-feeder-result: true
+  with-limit-violations: true
+  study-type: TRANSIENT
+  min-voltage-drop-proportional-threshold: 20
+```
+
+Available parameters in the short circuit API are stored in `com.powsybl.shortcircuit.ShortCircuitParameters`. They are all optional.
+
+**withLimitViolations**
```

Comment:
You must respect the format. Is it `with-limit-violations` or `withLimitViolations`?